docs/development/heat_dissipation.md

```diff
@@ -1,11 +1,92 @@
-# Heat dissipation
-
-::: warning TODO
-- Description of losses in power electronics
-    - Switching losses
-    - Conduction losses
-- Methods for cooling
-    - Passive using PCB
-    - Passive via heat sink
-    - Active (forced convection)
-:::
+# Losses and Heat
+
+In every real electrical system, flowing currents are generating heat inside the components. While most low energy devices do not generate siginificant amount of thermal waste, power electronics need to dissipate large amount of thermal energy.
+
+There are many different components on a board that contribute to the overall power loss, like resistors, traces, fuses, coil resistance and other ohmic resistances. In circuits with frequent switching, the impedance of capacitators and inductances can play a role as well, but won't be treated here.
+
+The main factor for heat generation in power electronics are the semi-conductors, mainly **MOSFETS** used for switching. In the following, switching and conduction losses for **MOSFETs** are explained and different ways to dissipate the heat are given.
+
+## Conduction loss
+
+Conduction loss appears in the steady-state phase when the system is not changing anymore after being switched on. The power can be calculated using
+
+$$ P_{cond} = r_{ds} \cdot I^2 $$
+
+where $r_{ds}$ is the internal gate to source resistance of the transistor, which can be obtained from the datasheet and $I$ is the current flowing through it. If the transitor is used in converters or else and is switched on and off frequently (e.g. a PWM [charge controller](../system/charge_controller.md)), the duty cycle (in percent) should be used to weigh the power loss:
+
+$$ P_{cond} = r_{ds} \cdot I^2 \cdot D$$
+
+To reduce conduction loss, a bigger transistor with a larger channel can be used to reduce $r_{ds}$. That also means a longer switching time because the capacitance in the transistor is greater, which increases the switching time and therefor the switching loss. In high frequency switching applications with a low duty cycle, it can be better to optimize for low switching time than low internal resistance.
```

Comment:
```suggestion
To reduce conduction loss, a bigger transistor with a larger channel can be used to reduce $r_{ds}$. That also means a longer switching time because the internal gate capacitance in the transistor is greater, which increases the switching time and therefore the switching loss. In high frequency switching applications with a low duty cycle, it can be better to optimize for low switching time than low internal resistance.
```